portfolio/pom.xml

```diff
@@ -17,19 +17,20 @@
   </properties>
 
   <dependencies>
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
+    </dependency>
```

Comment:
Out of curiosity, is there a reason this is moved (it used to be below javax.servlet)?

Reply:
it was part of my messy debugging process and I figured the order wouldn't do any harm so I never changed them back, but I can if its a formatting thing?